debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresMetricsIT.java

```diff
@@ -154,6 +154,8 @@ private void assertSnapshotMetrics() throws Exception {
         Assertions.assertThat(mBeanServer.getAttribute(getSnapshotMetricsObjectName(), "SnapshotRunning")).isEqualTo(false);
         Assertions.assertThat(mBeanServer.getAttribute(getSnapshotMetricsObjectName(), "SnapshotAborted")).isEqualTo(false);
         Assertions.assertThat(mBeanServer.getAttribute(getSnapshotMetricsObjectName(), "SnapshotCompleted")).isEqualTo(true);
+        Assertions.assertThat(mBeanServer.getAttribute(getSnapshotMetricsObjectName(), "SnapshotPaused")).isEqualTo(false);
+        Assertions.assertThat(mBeanServer.getAttribute(getSnapshotMetricsObjectName(), "SnapshotPausedDurationInSeconds")).isEqualTo(0L);
```

Comment:
I noticed we use the "SnapshotPaused" prefix here and in other places and I'm wondering if there is any concern with these metrics being confused with traditional snapshot-based metrics around running, aborted, and completed rather than using "IncrementalSnapshotPaused" or "AdhocSnapshotPaused".  

Also, we should also document these metrics in the documentation.

Reply:
@Naros good point. I was also thinking about it, but as we already have metrics for in incremental snapshot without this prefix (`SnapshotRunning`, `SnapshotDurationInSeconds` etc), I decided for consistency and use it without "Incremental" prefix. It would be probably even more confusing for the users if part of the metrics would start with "Incremental" and part not, although it's related to incremental metrics. 

As we work on 2.0 where we can introduce backward incomapatible changes, we may rename all the metrics related to incremental snapshot.

As for the documentation, I completely forgot on it. Will added. Thanks!

Reply:
@Naros documentation added. Looking more into JXM (and also signalling documentation), it looks like we mix traditional and incremental freely (use one JXM metric for both of them). Similar signalling documentation distinguish between `ad-hoc` snapshot and `incremental` snapshot, but e.g. in case of `stop-snapshot`, it speaks about `ad-hoc` snapshot and later on uses `incremental` snapshot. So I'm little but confused what is the right approach here. I included pause and resume signals under `Incremental snapshots` sub-chapter.

Reply:
@vjuranek ah, I didn't realize those others were related to incremental snapshot and didn't have a prefix, so I can see your point. Perhaps what we can take is a small action item to discuss with @jpechane when he returns if he feels we need to improve the naming of these metrics to provide a clear distinction and if so we can create a follow-up task for that.

As for the signal documentation itself, I would agree this sounds a bit messy.  I'll take a look at the documentation and I'll add some further comments on that commit if I feel they're necessary.

Reply:
Looking at what you did, I think I would agree that stop should be done similar to what you did for pause/resume.  I'll open a small follow-up for this and I'll take care of that tomorrow.